### PR TITLE
{lang} Python 2.7.11 CrayGNU 2016.03

### PR DESCRIPTION
**Pull request by gppezzi**

### CODE REVIEW

File: easybuild/easyconfigs/l/libreadline/libreadline-6.3-CrayGNU-2016.03.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+toolchain = {'name': 'CrayGNU', 'version': '2016.03'}
+toolchainopts = {'pic': True}
+
+sources = ['readline-%(version)s.tar.gz']
+source_urls = ['http://ftp.gnu.org/gnu/readline']
+
+patches = ['libreadline-%(version)s-bugfix.patch']
+
+dependencies = [('ncurses', '5.9')]
```
Comment (boegel) [inline easybuild/easyconfigs/l/libreadline/libreadline-6.3-CrayGNU-2016.03.eb]: @gppezzi we made the jump to `ncurses` 6.0 in `foss/2016a` and `intel/2016a`, maybe good to do it here too?

File: easybuild/easyconfigs/p/Python/Python-2.7.11-CrayGNU-2016.03.eb [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+# patch needed for building h5py
+patches = ['h5py-CrayGNU-Python-2.7.patch']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '5.9'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-CrayGNU-2016.03.eb]: @gppezzi 6.0

File: easybuild/easyconfigs/n/ncurses/ncurses-6.0-CrayGNU-2016.03.eb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+configopts = [
+    # default build
+    '--with-shared --enable-overwrite',
+    # the UTF-8 enabled version (ncursesw)
+    '--with-shared --enable-overwrite --enable-ext-colors --enable-widec --includedir=%(installdir)s/include/ncursesw/'
+]
+
+libs = ["form", "menu", "ncurses", "panel"]
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ["captoinfo", "clear", "infocmp", "infotocap", "ncurses5-config",
```
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.0-CrayGNU-2016.03.eb]: @gppezzi `ncurses6-config`

**Comment by gppezzi** [comment]
Test report by @gppezzi
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
dora102 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.7
See https://gist.github.com/3fd3a885ddc1a085caa0ffc6479cf896 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 9 (8 easyconfigs in this PR)
daint101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/5ea28e9b8194ed881d077b6f476a6a07 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
daint101 - Linux SLES 11_SP3, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/3cf85f5c62de80e326433a40c18f5ac5 for a full test report.

**Comment by boegel** [comment]
lgtm, going in, thanks @gppezzi!

**Pull request merged**